TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

```diff
@@ -114,101 +113,14 @@ kinematics.new WheelVelConstraint(PARAMS.maxWheelVel),
 
     public final Localizer localizer;
     public Pose2d pose;
-
```

Comment:
??

Reply:
what?

Reply:
Generally it's good to avoid changing whitespace in random places unrelated to your changes. I don't usually care too much about that, but any unneeded change in the quickstart has the potential to cause git conflicts for many downstream users.